common/smoke-test/smoke-tests.yml

```
@@ -49,9 +62,9 @@ jobs:
       - template: ../../eng/pipelines/templates/steps/common.yml
 
       - task: NodeTool@0
```

```suggestion
      - task: NodeTool@0
        inputs:
          versionSpec: $(NodeVersion)
          versionSpec: $(SmokeTestNodeVersion)
```
- template: ../steps/use-node-test-version.yml